### PR TITLE
New package: axel-2.4

### PR DESCRIPTION
**Pull request by pullmoll**
Doesn't compile on mips and mipsel.

### CODE REVIEW

File: srcpkgs/axel/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+revision=1
+wrksrc=axel-2.4
+build_style=gnu-configure
+short_desc="Download accelerator for FTP and HTTP"
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="GPL-2"
+homepage="https://axel.alioth.debian.org/"
+distfiles="${DEBIAN_SITE}/main/a/axel/axel_${version}.orig.tar.gz"
+checksum="359a57ab4e354bcb6075430d977c59d33eb3e2f1415a811948fa8ae657ca8036"
+only_for_archs="x86_64 x86_64-musl i686 i686-musl armv6hf armv6hf-musl armv7hf armv7hf-musl"
```
Comment (xtraeme) [inline srcpkgs/axel/template]: Please remove this line, it's pointless.

File: srcpkgs/axel/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+do_configure() {
+	./configure --prefix=/usr
```
Comment (xtraeme) [inline srcpkgs/axel/template]: If you don't pass `${configure_args}` the cross compilation will fail, I'd suggest you to remove `./configure` and rename `do_configure()` to `post_configure()`.

**Comment by xtraeme** [comment]
`/etc/axelrc` must be added to `conf_files` as well, and please disable stripping via configure.

**Comment by pullmoll** [comment]
Hmmm.. removing strip requires a dummy STRIP= in Makefile.settings, perhaps echo. I'll try.

File: srcpkgs/axel/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'axel'
+pkgname=axel
+version=2.4
+revision=1
+wrksrc=axel-2.4
```
Comment (xtraeme) [inline srcpkgs/axel/template]: This line is redundant, `wrksrc` by default is set to `${pkgname}-${version}`.

**Comment by xtraeme** [comment]
You should follow the standard commit log style:

```
<pkgname>: ...
```

**Comment by pullmoll** [comment]
Perhaps xlint can be improved to detect redundant wrksrc?

**Comment by xtraeme** [comment]
I agree // cc @chneukirchen

**Pull request merged**